lib/meta.js

```diff
@@ -1,35 +1,28 @@
 'use strict';
 
-const Promise = require('bluebird');
-const exec = require('child_process').exec;
+const exec = require('child_process').execSync;
 
 /**
   * Gathers all require-able Node modules
   * @module meta
   *
-  * @returns {Promise<object>} A Promise that returns an object containing node_module names and meta information about those node_modules, including their `package.json` information
+  * @returns {Array} A Array of `require`-able Node modules as an object containing their paths and their `package.json` information
 **/
-var meta = function () {
-  return new Promise(function (resolve, reject) {
-    exec('npm ls --json --depth=0 --long', function (err, stdout) {
-      var results;
-      var modules = [];
+const meta = function () {
+  var npmModules = exec('npm ls --json --depth=0 --long');
+  var results;
+  var modules = [];
 
-      if (err) {
-        reject(err);
-      }
+  results = JSON.parse(npmModules);
 
-      results = JSON.parse(stdout);
-      Object.keys(results.dependencies).forEach(function (dep) {
-        dep = results.dependencies[dep];
-        modules.push({
-          path: dep.path,
-          package: dep
-        });
-      });
-      resolve(modules);
+  Object.keys(results.dependencies).forEach(function (dep) {
```

Comment:
Yes, it would. .filter does arrays of objects too:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter